src/main.jsx

```diff
@@ -48,7 +49,13 @@ if (__TARGET__ === 'mobile') {
 let store, client, history, lang, root
 
 const initRender = () => {
-  if (handleOAuthResponse()) return
+  if (handleOAuthResponse()) {
+    render(
+      <Spinner size="xxlarge" middle={true} />,
+      document.querySelector('[role=application]')
+    )
+    return
+  }
```

Comment:
Should we add it to the Home app too? 

Also we spoke about adding a text like "Finalization in progress"

Reply:
This will be added to the home application too (after this review).

Now I added the text without localization.
Adding localization may make the load time longer in this case and I would have to duplicate the loading of translations. Do you think I should do it ?

Reply:
You should have describe that if the previous version of the PR. I would not asked about the text. 

Let's remove it instead of having in a the wrong language

Reply:
Updated